orchagent/pfcactionhandler.cpp

```diff
@@ -133,6 +133,86 @@ void PfcWdActionHandler::updateWdCounters(const string& queueIdStr, const PfcWdQ
     m_countersTable->set(queueIdStr, resultFvValues);
 }
 
+PfcWdAclHandler::PfcWdAclHandler(sai_object_id_t port, sai_object_id_t queue,
+        uint8_t queueId, shared_ptr<Table> countersTable):
+    PfcWdActionHandler(port, queue, queueId, countersTable)
+{
+    acl_table_type_t table_type = ACL_TABLE_L3;
+
+    string portstr = sai_serialize_object_id(port);
+    string queuestr = sai_serialize_object_id(queue);
+    m_strTable = "Table_" + portstr + "_" + queuestr;
```

Comment:
the Table name should be PFC_INGRESS_DROP_TC_X. It does not need to have port name, since this table will be able to bind to multiple ports.